clang/include/clang/Driver/ToolChain.h

```diff
@@ -95,18 +95,21 @@ class ToolChain {
 
   enum CXXStdlibType {
     CST_Libcxx,
-    CST_Libstdcxx
+    CST_Libstdcxx,
+    CST_Stl,
```

Comment:
Could this be Msvcstl or something? Just Stl seems a pretty generic name.

Reply:
> Could this be Msvcstl or something? Just Stl seems a pretty generic name.

The C++ standard never says anything about stl. STL is not an official term in fact.

Neither libc++ nor libstdc++ are not pretty generic. Why are they not called llvmlibc++ or gnulibstdc++?

Reply:
I think most C++ developers understand STL as a generic term for the standard library rather than for Microsoft's implementation specifically.

`-stdlib=stl` could be confusing; `-stdlib=msstl` would be much clearer. (And the enumeration should be named to match.)